aiida/cmdline/commands/exportfile.py

```diff
@@ -41,20 +41,33 @@ def cli(self, *args):
     help='Export the given groups by pk')
 @click.option('-g', '--group_names', multiple=True, type=str,
     help='Export the given groups by group name')
-@click.option('-P', '--no-parents', is_flag=True, default=False,
-    help='Store only the nodes that are explicitly given, without exporting the parents')
-@click.option('-O', '--no-calc-outputs', is_flag=True, default=False,
-    help='If a calculation is included in the list of nodes to export, do not export its outputs')
+@click.option('-IF', '--input_forward', is_flag=True, default=False,
```

Comment:
We should try to maintain the convention of only using single letters for single-letter-options.

Reply:
yes, but it seems that there are overlaps and many of the single letters are already reserved. For example which letter would you give to --call_reversed and  to --create_reversed flags. Furthermore, I would like the flags that override the default graph traversal for the export to be as uniform (among them) as possible.

Reply:
You could use `-I`, `-C`, `-R` and `-X` for example. Note that they don't overlap with any of the other flags in this command, nor are they necessarily less intuitive, specially given the long form is available. Which also the long form options should not use underscores, i.e. `--create_reversed` should be `--create-reversed`

Reply:
Please also consider adding *no* single-letter version.
These are just abbreviations, which make sense for the options that appear in many places (and people will thus memorize their meaning)

Reply:
Agreed, although they have another function besides easy memorization. If an option is used very often, having a short-hand single letter version is a lot faster to use, even with autocomplete functionality

Reply:
For reference the POSIX cli flag convention that we maintain: https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html